automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -315,7 +330,11 @@ trait AzureBillingBeforeAndAfter extends FixtureAnyFreeSpecLike with BeforeAndAf
 
     try
       testCode(response)
-    finally
+    catch {
```

Comment:
Is this still worth doing given the issue with that ghost test exiting without a trace?

Reply:
Yeah, I want to be extra verbose in logging until we track it down

Reply:
If we find the test logging is too noisy, we can slim it down. However, we seem to have a steady stream of PRs adding various one-line logging message for debugging, so I figure it can't hurt.